README.md

```diff
@@ -53,6 +53,9 @@ There are two ways you can install xFormers locally:
 
   ```bash
   git clone git@github.com:facebookresearch/xformers.git
+  git submodule update --init --recursive
```

Comment:
nice, I was chacking that when seeing that xformers now has two submodules, perfect. Thanks